src/views/SetKeys.js

```diff
@@ -122,22 +124,34 @@ class SetKeys extends React.Component {
     })
   }
 
+  randomHex(len) {
```

Comment:
Looks like [web3 has a function for this](https://web3js.readthedocs.io/en/1.0/web3-utils.html#randomhex). Supposedly "cryptographically strong". We should use that instead of rolling our own.

Reply:
So I tried this out, and it turns out the function is pretty borked; it doesn't generate strings of consistent length >.<. How important is 'cryptographic strength' here do you think?

https://github.com/ethereum/web3.js/issues/1490

Reply:
*sigh*

Just slap a `//TODO use web3.utils.randomHex, see web3.js#1490` in there and call it a day. web3.js is such a shitshow rn. ):